### PR TITLE
Update Network.cpp

### PR DESCRIPTION
**Pull request by yonatanmg**
-removed the randomizeMatrix since the matrices will be populated with actual data anyway
-loop for initializing matrix of previous network has ignore statements due to formatting issues. consider not printing "w/b [index]" to output file for cleaner code.

### CODE REVIEW

**Comment by patelami3431** [comment]
Yeah, you're right about randomizeMatrix. Don't know why I did that!
I can remove the line (w/b) [index] from the file but I added that so we know what we are looking for when we open the file.

File: Network Class/Network.cpp [hunk 1]
```diff
@@ -138,6 +138,39 @@ void Network::updateWeightsAndBiases()
...
+	
+	bool correct = true;
+	// to write: check if activations[lastInd] matches expectedVal at index and set correct accordingly
+
+	// Matrix costPrime = activations[lastInd] - expectedValues;
+	// above statement needs expectedValues to be a matrix to work
+	// consider the implications of having getAt return a (j x 1) matrix instead of a vector
+	Matrix costPrime; costPrime.set_size(lastSize, 1);
+	for (int i = 0; i < lastSize; i++
+		 costPrime(i,0) = activations[lastInd](i,0) - expectedValues[i];
```
Comment (patelami3431) [inline Network Class/Network.cpp:164]: costPrime() has to be a function, outside the class. Cost Function is something the programmers in future, or even we, will be changing!

File: Network Class/Network.cpp [hunk 2]
```diff
@@ -138,6 +138,39 @@ void Network::updateWeightsAndBiases()
...
+	int lastInd = numLayers - 1;
+	int lastSize = layerSizes[lastInd];
+	Vector expectedValues = getAt<int>(expectedValuesInfile, index);
+	
+	bool correct = true;
+	// to write: check if activations[lastInd] matches expectedVal at index and set correct accordingly
+
+	// Matrix costPrime = activations[lastInd] - expectedValues;
+	// above statement needs expectedValues to be a matrix to work
+	// consider the implications of having getAt return a (j x 1) matrix instead of a vector
```
Comment (patelami3431) [inline Network Class/Network.cpp]: We can overload getAt(), so we can use it to assign the weightedInputs and the matrices if needed!

File: Network Class/Network.cpp (hunk 1 above)
Comment (yonatanmg) [reply Network Class/Network.cpp:164]: i've added the modified code for using a cost prime func in a comment in the newest commit and will be changing it when we have a cost prime func written.

Comment (patelami3431) [reply Network Class/Network.cpp]: Alright! And also, the backprop will always return a true. Make sure to change that too

Comment (patelami3431) [reply Network Class/Network.cpp]: Alright! And also, the backprop will always return a true. Make sure to change that too

**Pull request merged**